src/alchemlyb/estimators/mbar_.py

```diff
@@ -187,9 +211,27 @@ def fit(self, u_nk):
         self.theta_ = pd.DataFrame(
             out["Theta"], columns=self._states_, index=self._states_
         )
+        if compute_entropy_enthalpy:
+            self._delta_h_ = pd.DataFrame(
+                out["Delta_u"], columns=self._states_, index=self._states_
+            )
+            self._d_delta_h_ = pd.DataFrame(
+                out["dDelta_u"], columns=self._states_, index=self._states_
+            )
+            self._delta_sT_ = pd.DataFrame(
```

Comment:
I'm happy for this value to be s*T. Is the mbar output s or s*T?

Reply:
Well the mbar output is reduced entropy, so S/k or s*T/(k*T) so it's consistent.

Reply:
I know that the unit is the same but I worried about the magnitude.
The G=H-sT so if the `Delta_s` is s then the `Delta_f` should be the same as `Delta_u`-`Delta_s` * T. If Delta_s is sT, then `Delta_f` should be `Delta_u`-`Delta_s`

Reply:
Yes I agree, so I'm changing the variable names to make that clear.  I'll do whatever the maintainers agree on, maybe @mrshirts or @orbeckst have an opinion?

**Summary**: Entropy has different units and which need to be handled properly. While the units of enthalpy and free energy are `attr['energy_units'] = 'kT', 'kcal/mol' or 'kJ/mol'`, the equivalent for entropy would then be `'k', 'kcal/mol/K' or 'kJ/mol/K'`. The downside of this difference in units would require either having copies of the units.py functions for entropy specifically or add a series of if statements to the functions in units.py to detect and handle entropy units. Also, it's awkward to say that dimensionless entropy has units of `'k'`

**Alternative**: By having the entropy matrix represent S*T (denoted with the variable name `delta_sT_`) the object can retain the same units as enthalpy and free energy and use the same unit conversion functions, making everything cleaner internally, and avoid the awkward unit "definition" of `'k'` for S.

Reply:
Thanks. I'm merging this PR. I have checked the numerical values and can verify that Delta_s is not s but rather s*T. So it is in the unit of kT, kJ/mol and kcal/mol.